js/importsms.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Library to help with importing an SMS game config from JSON to Twine data.
+ */
+
+'use strict';
+
+var importsms = function() {
+
+  /**
+   * Import stories from a JSON string.
+   *
+   * @param filename
+   *   Filename
+   * @param strData
+   *   JSON string of story data
+   * @return Number of stories imported
+   */
+  function importStory(filename, strData) {
+    var data,
+        allStories,
+        allPassages,
+        story,
+        storyConfig,
+        pIndex,
+        passages,
+        keys,
+        i;
+
+    data = JSON.parse(strData);
+
+    allStories = StoryCollection.all();
```

Comment:
Why is it necessary to run the `StoryCollection.all()` function to create stories?